LayoutTests/svg/css/svg-write-resources-null-maskImage-crash-expected.html

```diff
@@ -0,0 +1,5 @@
+<svg>
+    <g style="mask-image: none, url()">
+    </g>
```

Comment:
Not sure you need the `<g>` tag in the reference fwiw (or even if the `<svg>` tag is needed)

Can you try removing them?